### PR TITLE
Updated verbiage - Correctly detected closing </body> tag.

### PR DESCRIPTION
**Pull request by garretthyder**
Updated the verbiage on successful detection of </body> tag to clarify it's not an error.
New Verbiage - `Correctly detected closing </body> tag.`

Note: I wasn't sure if the following feature files should have been updated;
- `features/fail-http-code.feature`
- `features/fail-uncaught-fatal.feature`
- `features/pass.feature`
*If they shouldn't be updated I can remove from the PR.

Thanks

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -85,5 +85,8 @@ Both installation methods expose the `wp core safe-update` WP-CLI command, which
 
 == Changelog ==
 
+### 0.1.1 (??? ??, 2018) ###
+* Updated 'closing </body> tag' detection message to clarify it's a success.
```
Comment (danielbachhuber) [inline readme.txt]: 0.1.0 is still unreleased, so you don't need to include this changelog entry.

**Review by danielbachhuber — COMMENTED** [review]
Can you merge master to fix the failing build with #30?

File: readme.txt (hunk 1 above)
Comment (garretthyder) [reply readme.txt]: Thanks, should I remove fully or just move under 0.1.0?

**Comment by garretthyder** [comment]
Alright, thanks for #30 the tests are now passing. Let me know what you want to do on the Changelog and we'll wrap this up. Cheers

File: readme.txt (hunk 1 above)
Comment (danielbachhuber) [reply readme.txt]: I've removed in 9e110ae0921c87e0442fe13b109c00097f2761e3

Comment (garretthyder) [reply readme.txt]: Thanks @danielbachhuber

**Comment by danielbachhuber** [comment]
Thanks for your contribution!

**Comment by garretthyder** [comment]
All the best

**Pull request merged**